src/Microsoft.Azure.ServiceBus/MessageReceivePump.cs

```diff
@@ -76,7 +79,7 @@ bool ShouldRenewLock()
                 catch (Exception exception)
                 {
                     MessagingEventSource.Log.MessageReceivePumpTaskException(this.messageReceiver.ClientId, string.Empty, exception);
-                    this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Receive));
+                    this.registerHandlerOptions.RaiseExceptionReceived(new ExceptionReceivedEventArgs(exception, ExceptionReceivedEventArgsAction.Receive, namespaceName));
```

Comment:
if we are passing namespace, I feel we should definitely pass the entity name as well. Could you include `messageReceiver.Path` as well